### PR TITLE
minicli: Updates #731. Start addressing minicli Header uniformity.

### PR DESCRIPTION
**Pull request by jasoncontrib**
Opening a PR now so we can start discussing specifics. Need to fix minitests to match once we've decided on column names (and units?).

### CODE REVIEW

**Comment by jasoncontrib** [comment]
Good point, for some reason I was (mistakenly) worried about not appending to the tabular unless the function was going to return cleanly. Just needs a `columns[i] = header` so that the returned header uses the full column names and not the incomplete column names.

**Comment by jasoncontrib** [comment]
We may have to have some ugliness in the code (or an extra loop) after all, to avoid the corner case where an exact match is found AFTER a partial match, e.g.

```
minimega$ .filter name~host .col namespace,name,st vm info
Error (tinkerpad): ambiguous column `name`
minimega$ .filter name~host .col name,namespace,st vm info
host      | name  | namespace | state
tinkerpad | host1 |           | RUNNING
tinkerpad | host2 |           | RUNNING
```

File: src/minicli/builtins.go [hunk 1]
```diff
@@ -280,6 +281,34 @@ func parseFilter(s string) (filter, error) {
...
+		if foundI >= 0 {
+			// collision
+			return 0, fmt.Errorf("ambiguous column `%s`", column)
+		}
+
+		foundI = i
+	}
+
+	if foundI >= 0 {
+		return foundI, nil //TODO
```
Comment (jcrussell) [inline src/minicli/builtins.go]: What is TODO for?

Comment (jasoncontrib) [inline src/minicli/builtins.go]: Oops, forgot to remove the TODO. It does work and passes the `builtins` minitest with apropos / partial matching additions.

**Pull request merged**